### PR TITLE
Fix local docker compose

### PR DESCRIPTION
**Pull request by Rados13**

### CODE REVIEW

**Review by LVala — APPROVED** [review]

File: .github/workflows/test_build_and_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+      - name: Deploy docker compose to a pre-configured server
+        id: deploy
+        uses: TapTap21/docker-remote-deployment-action@v1.1
+        with:
+          remote_docker_host: ${{ secrets.SANDBOX_HOST }}
+          ssh_private_key: ${{ secrets.SSH_PRIV_KEY }}
+          ssh_public_key: ${{ secrets.SSH_PUB_KEY }}
+          stack_file_name: ${{ env.COMPOSE_FILE_NAME }}
+          args: -p ${{ env.PROJECT }} --env-file .env up -d --remove-orphans
```
Comment (roznawsk) [inline .github/workflows/test_build_and_deploy.yml:55]:
```suggestion
          args: -p ${{ env.PROJECT }} --env-file .env up -d --remove-orphans

```

File: lib/videoroom/notifier.ex [hunk 2]
```diff
@@ -12,14 +12,8 @@ defmodule Videoroom.Notifier do
...
-
-    {:ok, nil}
-  end
-
-  @impl true
-  def handle_info({:DOWN, _ref, :process, _object, _reason}, state) do
-    Logger.warning("Jellyfish Notifier exited")
-    {:noreply, state}
+    {:ok, %{}}
   end
```
Comment (roznawsk) [inline lib/videoroom/notifier.ex:18]: Why has this been removed? If Jellyfish stops working for some reason and the Notifier connection is closed is there any way to acknowledge it?

**Review by roznawsk — CHANGES_REQUESTED** [review]

File: lib/videoroom/notifier.ex (hunk 2 above)
Comment (Rados13) [reply lib/videoroom/notifier.ex:18]: I think we could acknowledge it in the Meeting process.

Comment (roznawsk) [reply lib/videoroom/notifier.ex:18]: Ok, could you please add the monitoring and logging of the notifier in the meeting?

**Review by kamil-stasiak — APPROVED** [review]

**Review by roznawsk — APPROVED** [review]

**Review by blazpie — APPROVED** [review]

**Pull request merged**